src/lxc/confile.c

```diff
@@ -5463,6 +5715,8 @@ int lxc_list_net(struct lxc_conf *c, const char *key, char *retv, int inlen)
 	switch (netdev->type) {
 	case LXC_NET_VETH:
 		strprint(retv, inlen, "veth.pair\n");
+		strprint(retv, inlen, "ipv4.route\n");
```

Comment:
@stgraber @brauner I noticed that I had forgotten to add the new route params to this section of the code.

In doing so it seemed to me that `ipv4.route` and `ipv6.route` params when they are only applicable to `veth` type seems to be a bit of a "code smell", and that it would be more consistent to be `veth.ipv4.route` and `veth.ipv6.route` respectively.

Happy for me to move these into the `veth` section?

Reply:
yeah, probably a good idea